doc/tutorials/analyze-ice-sheet-volume.md

```diff
@@ -0,0 +1,69 @@
+# Calculate the volume of the Greenland ice sheet
+
+The **Processing Toolbox** in QGIS provides a collection of tools and
+pre-written algorithms that allow the user to perform a wide variety of raster
+and vector data analyses. See the [QGIS
+manual](https://docs.qgis.org/3.16/en/docs/user_manual/processing/toolbox.html)
+for complete documentation on the **Processing Toolbox**.
+
+In this tutorial, the user will utilize the **Processing Toolbox** with data in
+QGreenland to calculate the Greenland ice sheet's volume. In doing so, the user
+will become familiar with the how to use the **Processing Toolbox** to perform
+geospatial analysis.
+
+
+```{warning}
```

Comment:
Maybe a `{note}` instead? :shrug:

Reply:
I figured warning made sense here but could just as easily be a note.